src/multi.jl

```diff
@@ -170,8 +179,7 @@ function populate_convhull_extreme_values(m::PODNonlinearModel, discretization::
 end
 
 """
-    TODO: docstring
-    method for general multilinear term
+    Extreme value for gneral multilinear terms Efficient implementation
```

Comment:
general

Reply:
This is taken care of in the most recent commit.